app/models/certificate.server.ts

```diff
@@ -4,15 +4,18 @@ import { prisma } from '~/db.server';
 
 export type { Certificate } from '@prisma/client';
 
-export function getIssuedCertificateByUsername(username: Certificate['username']) {
+export async function getCertificateByUsername(username: Certificate['username']) {
```

Comment:
Because you don't `await` in this function, and return the `Promise` directly, you don't need `async` here.

Reply:
I did that before, but I did receive a warning that it might turn into an async
![image](https://user-images.githubusercontent.com/32577022/228306519-4ad62acc-ba69-4420-bb13-e78f5333b6a3.png)